packages/cli/src/util/dev/server.ts

```diff
@@ -1278,7 +1278,21 @@ export default class DevServer {
     }
 
     const method = req.method || 'GET';
-    this.output.debug(`${chalk.bold(method)} ${req.url}`);
+    const space = method === 'GET' ? '  ' : ' '; // make GET and POST take up the same width
+    this.output.debug(`${chalk.bold(method)}${space}${req.url}`);
```

Comment:
The `method` and `space` definitions can essentially be combined.

```suggestion
    const method = (req.method || 'GET').padEnd(4);
    this.output.debug(`${chalk.bold(method)} ${req.url}`);
```

I dunno, but I think this is a bit more readable since I don't need to count spaces. Plus the above supports `PUT` just in case.

Reply:
I didn't realize `padEnd` was built-in. Sounds good to me! [`bf390cc` (#9411)](https://github.com/vercel/vercel/pull/9411/commits/bf390cc13b8b2e56d9b1e5b2b02931dfba16f81c)